packages/devtools/analysis_options.yaml

```diff
@@ -1,9 +1,11 @@
+# Note: when updating this file, update devtools_server/analysis_options.yaml to match.
+
 # Start with a base-line of the package:pedantic lints.
 include: package:pedantic/analysis_options.yaml
 
 analyzer:
-#  strong-mode:
-#    implicit-casts: false
+  #  strong-mode:
```

Comment:
I'm guessing this indent change was accidental

Reply:
No this was on purpose to make the two analysis options files identical. These two comments seemed to be the only ones sitting on the margin, so I changed this file to match the pattern.